pkg/client/clientset_generated/clientset/typed/kops/v1alpha2/fake/fake_keyset.go

```diff
@@ -0,0 +1,120 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package fake
+
+import (
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	labels "k8s.io/apimachinery/pkg/labels"
+	schema "k8s.io/apimachinery/pkg/runtime/schema"
+	types "k8s.io/apimachinery/pkg/types"
+	watch "k8s.io/apimachinery/pkg/watch"
+	testing "k8s.io/client-go/testing"
+	v1alpha2 "k8s.io/kops/pkg/apis/kops/v1alpha2"
+)
+
+// FakeKeysets implements KeysetInterface
+type FakeKeysets struct {
+	Fake *FakeKopsV1alpha2
+	ns   string
+}
+
+var keysetsResource = schema.GroupVersionResource{Group: "kops", Version: "v1alpha2", Resource: "keysets"}
+
+var keysetsKind = schema.GroupVersionKind{Group: "kops", Version: "v1alpha2", Kind: "Keyset"}
+
+func (c *FakeKeysets) Create(keyset *v1alpha2.Keyset) (result *v1alpha2.Keyset, err error) {
```

Comment:
Was this generated?  If not we need code comments please

Reply:
Generated